agent/config/config_windows.go

```diff
@@ -24,16 +24,40 @@ import (
 
 const (
 	defaultCredentialsAuditLogFile = `log\audit.log`
-	httpPort                       = 80
+	// When using IAM roles for tasks on Windows, the credential proxy consumes port 80
+	httpPort = 80
```

Comment:
lint only enforces this for public things (consts, variables, structs, struct fields), since there's no godoc on private ones.  I can change this if you feel strongly.